packages/cursorless-neovim-e2e/src/suite/recorded.neovim.test.ts

```diff
@@ -71,9 +71,8 @@ async function openNewTestEditor(
   await client.command(":enew");
 
   if (!openBeside) {
-    // close all the other buffers
-    // @see: https://stackoverflow.com/questions/4545275/vim-close-all-buffers-but-this-one
-    await client.command(":BufOnly!");
+    // close all the other buffers (<C-^> is needed because e# fails on unnamed buffers)
```

Comment:
what does `e#` mean in this context? `execute`?

Reply:
e is edit https://neovim.io/doc/user/editing.html#_2.-editing-a-file
# is 'alternate buffer' as in last used buffer, but can't be used to target unnamed buffers. 

because this works by closing all buffers first, we then want to reopen the last one that we triggered the command from, but because it's potentially unnamed we can't use :e#, thus the different approach.